editor/editor_node.cpp

```diff
@@ -3588,6 +3588,13 @@ void EditorNode::set_current_scene(int p_idx) {
 	call_deferred(SNAME("_set_main_scene_state"), state, get_edited_scene()); // Do after everything else is done setting up.
 }
 
+void EditorNode::setup_color_picker(ColorPicker *picker) {
+	int default_color_mode = EDITOR_GET("interface/inspector/default_color_picker_mode");
+	int picker_shape = EDITOR_GET("interface/inspector/default_color_picker_shape");
+picker->set_color_mode((ColorPicker::ColorModeType)default_color_mode);
+	picker->set_picker_shape((ColorPicker::PickerShapeType)picker_shape);
```

Comment:
```suggestion
	int default_color_mode = EDITOR_GET("interface/inspector/default_color_picker_mode");
	int picker_shape = EDITOR_GET("interface/inspector/default_color_picker_shape");
	picker->set_color_mode((ColorPicker::ColorModeType)default_color_mode);
	picker->set_picker_shape((ColorPicker::PickerShapeType)picker_shape);
```

Reply:
For some reason clang failed that original line, so I tried the suggested change just to see what would happen.